redis/Makefile

```diff
@@ -16,3 +16,18 @@ gen-dev-enablement-redis-shared:
 		--workdir=/opt/manifests \
 		--entrypoint=/bin/sh \
 		alpine/helm ./gen-yaml/gen.sh
+
+.PHONY: gen-bank-account-tokenisation-redis-shared
+gen-bank-account-tokenisation-redis-shared:
+	docker run -ti --rm \
+		--volume $${PWD}:/opt/manifests \
+		--env BITNAMI_REDIS_RELEASE=${BITNAMI_REDIS_RELEASE} \
+		--env NS="payment-platform-banking" \
+		--env NAME="bank-account-tokenisation-redis" \
+		--env OPSLEVEL_APP_DESCRIPTION="cache for opslevel-k8s-deployer" \
```

Comment:
It needs description suiting your service :slightly_smiling_face:

Reply:
Updated the description.

Reply:
Need to regenerate, wait a sec

Reply:
Ok, should be all good now : )

Reply:
LGTM, but I'm afraid you'll have to regenerate it anyway when https://github.com/utilitywarehouse/shared-kustomize-bases/pull/6 will be merged.

Reply:
Will keep an eye out for that